.github/workflows/main.yml

```diff
@@ -1,20 +1,19 @@
 name: Build App
 
-on: 
+on:
   push:
-    branches: 
+    branches:
       - master
```

Comment:
Does this mean it will only run when we push (or merge) something to master? Should we also include it on PR branches?

Reply:
This PR should not be making any changes to any kind of logic; all that is being changed in this PR is formatting :+1:  I added the Prettier config, updated the contribution guidelines in the README and ran Prettier.